docs/using/how-it-works.md

```diff
@@ -22,7 +22,10 @@ and the grain occur over the Cap'n Proto WebSession format.  With existing
 applications, the Sandstorm HTTP bridge is used to translate between Cap'n
 Proto and HTTP.
 
-```graphviz dot communication_overview_http_app.svg
+![Diagram with HTTP Bridge](communication_overview_http_app.svg)
+
+<!--
```

Comment:
Might want to add a comment here explaining why this is commented out and how it relates to the svg.

Reply:
Done.